### ISSUE TITLE
ERROR: /bin/bash: line 1: 12830 Segmentation fault  

### ISSUE THREAD

**Issue by soporteCluster** [tensorflow/tensorflow#21687]
Hi,
i'm trying to install tensorflow with CUDA support and I have the following issue whe i execute
bazel build --config=opt --config=cuda //tensorflow/tools/pip_package:build_pip_package --verbose_failures

ERROR:

/bin/bash: line 1: 12830 Segmentation fault      bazel-out/host/bin/tensorflow/create_tensorflow.python_api --root_init_template=tensorflow/api_template.__init__.py --apidir=bazel-out/k8-opt/genfiles/tensorflow --apiname=tensorflow --apiversion=1 --package=tensorflow.python --output_package=tensorflow bazel-out/k8-opt/genfiles/tensorflow/__init__.py bazel-out/k8-opt/genfiles/tensorflow/app/__init__.py bazel-out/k8-opt/genfiles/tensorflow/bitwise/__init__.py bazel-out/k8-opt/genfiles/tensorflow/compat/__init__.py bazel-out/k8-opt/genfiles/tensorflow/data/__init__.py bazel-out/k8-opt/genfiles/tensorflow/debugging/__init__.py bazel-out/k8-opt/genfiles/tensorflow/distributions/__init__.py bazel-out/k8-opt/genfiles/tensorflow/dtypes/__init__.py bazel-out/k8-opt/genfiles/tensorflow/errors/__init__.py bazel-out/k8-opt/genfiles/tensorflow/feature_column/__init__.py bazel-out/k8-opt/genfiles/tensorflow/gfile/__init__.py bazel-out/k8-opt/genfiles/tensorflow/graph_util/__init__.py bazel-out/k8-opt/genfiles/tensorflow/image/__init__.py bazel-out/k8-opt/genfiles/tensorflow/io/__init__.py bazel-out/k8-opt/genfiles/tensorflow/initializers/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/activations/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/applications/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/applications/densenet/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/applications/inception_resnet_v2/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/applications/inception_v3/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/applications/mobilenet/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/applications/nasnet/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/applications/resnet50/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/applications/vgg16/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/applications/vgg19/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/applications/xception/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/backend/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/callbacks/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/constraints/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/datasets/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/datasets/boston_housing/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/datasets/cifar10/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/datasets/cifar100/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/datasets/fashion_mnist/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/datasets/imdb/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/datasets/mnist/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/datasets/reuters/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/estimator/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/initializers/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/layers/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/losses/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/metrics/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/models/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/optimizers/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/preprocessing/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/preprocessing/image/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/preprocessing/sequence/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/preprocessing/text/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/regularizers/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/utils/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/wrappers/__init__.py bazel-out/k8-opt/genfiles/tensorflow/keras/wrappers/scikit_learn/__init__.py bazel-out/k8-opt/genfiles/tensorflow/layers/__init__.py bazel-out/k8-opt/genfiles/tensorflow/linalg/__init__.py bazel-out/k8-opt/genfiles/tensorflow/logging/__init__.py bazel-out/k8-opt/genfiles/tensorflow/losses/__init__.py bazel-out/k8-opt/genfiles/tensorflow/manip/__init__.py bazel-out/k8-opt/genfiles/tensorflow/math/__init__.py bazel-out/k8-opt/genfiles/tensorflow/metrics/__init__.py bazel-out/k8-opt/genfiles/tensorflow/nn/__init__.py bazel-out/k8-opt/genfiles/tensorflow/nn/rnn_cell/__init__.py bazel-out/k8-opt/genfiles/tensorflow/profiler/__init__.py bazel-out/k8-opt/genfiles/tensorflow/python_io/__init__.py bazel-out/k8-opt/genfiles/tensorflow/quantization/__init__.py bazel-out/k8-opt/genfiles/tensorflow/resource_loader/__init__.py bazel-out/k8-opt/genfiles/tensorflow/strings/__init__.py bazel-out/k8-opt/genfiles/tensorflow/saved_model/__init__.py bazel-out/k8-opt/genfiles/tensorflow/saved_model/builder/__init__.py bazel-out/k8-opt/genfiles/tensorflow/saved_model/constants/__init__.py bazel-out/k8-opt/genfiles/tensorflow/saved_model/loader/__init__.py bazel-out/k8-opt/genfiles/tensorflow/saved_model/main_op/__init__.py bazel-out/k8-opt/genfiles/tensorflow/saved_model/signature_constants/__init__.py bazel-out/k8-opt/genfiles/tensorflow/saved_model/signature_def_utils/__init__.py bazel-out/k8-opt/genfiles/tensorflow/saved_model/tag_constants/__init__.py bazel-out/k8-opt/genfiles/tensorflow/saved_model/utils/__init__.py bazel-out/k8-opt/genfiles/tensorflow/sets/__init__.py bazel-out/k8-opt/genfiles/tensorflow/sparse/__init__.py bazel-out/k8-opt/genfiles/tensorflow/spectral/__init__.py bazel-out/k8-opt/genfiles/tensorflow/summary/__init__.py bazel-out/k8-opt/genfiles/tensorflow/sysconfig/__init__.py bazel-out/k8-opt/genfiles/tensorflow/test/__init__.py bazel-out/k8-opt/genfiles/tensorflow/train/__init__.py bazel-out/k8-opt/genfiles/tensorflow/train/queue_runner/__init__.py bazel-out/k8-opt/genfiles/tensorflow/user_ops/__init__.py
Target //tensorflow/tools/pip_package:build_pip_package failed to build
INFO: Elapsed time: 4401.213s, Critical Path: 251.23s
INFO: 7900 processes: 7900 local.
FAILED: Build did NOT complete successfully

OS: CentOS 6.2
Tensorflow installed from source.
Tensorflow version: 1.10
Python version: 2.7
Bazel version: 0.16.0
GCC version: 6.3.0
CUDA 8.0
cuDNN: 6.0
GPU: GeForce GTX 770

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by rohan100jain** [comment]
Can you please list out step by step what you did? So that we can try and reproduce it and then debug?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rohan100jain: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
